src/translate_c.zig

```diff
@@ -2688,16 +2688,15 @@ fn transInitListExprVector(
 ) TransError!Node {
     _ = ty;
     const qt = getExprQualType(c, @ptrCast(*const clang.Expr, expr));
-    const vector_type = try transQualType(c, scope, qt, loc);
     const init_count = expr.getNumInits();
 
     if (init_count == 0) {
-        return Tag.container_init.create(c.arena, .{
-            .lhs = vector_type,
-            .inits = try c.arena.alloc(ast.Payload.ContainerInit.Initializer, 0),
-        });
+        return Tag.undefined_literal.init();
```

Comment:
This should be `@splat(num_elements, zero)`.

Reply:
Oof, good catch - that could have led to some ugly bugs. Updated.